13.technical-support/02.Adware/docs.en.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Adware/Viruses
+published: true
+taxonomy:
+    category:
+        - docs
+---
+
+If you suspect your computer is infected by malware or viruses, first of all you need to check your computer with free utilities from leading antiviruses.
+
+**Free antivirus utility**
+
+[Kaspersky Virus Removal Tool](https://www.kaspersky.ru/downloads/thank-you/free-virus-removal-tool)
+
+[Dr.Web CureIt! ](http://www.freedrweb.com/cureit/?lng=en)
+
+If this check with antivirus utilities revealed nothing, but the computer still behaves suspiciously, try to test it for adware, which can implement its advertising on any pages that you open.
+
+## How to clean your system from Adware?
+
+In the Internet, you can find a lot of applications that are aimed at fighting this problem. You can use any utility you like. But we strongly recommend using **[AdwareCleaner](https://downloads.malwarebytes.com/file/adwcleaner)** from **Malwarebytes**. This application is quite easy to use and you can test it by yourself.
```

Comment:
On the Internet, you can find a lot of applications that are aimed at fighting this problem. You can use any utility you like. But we strongly recommend using **[AdwCleaner](https://downloads.malwarebytes.com/file/adwcleaner)** by **Malwarebytes**. This application is quite easy to use and you will be able to handle it by yourself.